serverless.yml

```diff
@@ -284,10 +285,10 @@ functions:
               - name: request.header.origin
 
   get_dag_metadata_handler:
-    handler: handlers/get_dag_metadata_handler.handle
+    handler: handlers/dag_metadata_handler.handle_get
```

Comment:
Note: the `get_dag_metadata_handler.py` file was renamed to `dag_metadata_handler.py` to accomodate both the `get` and `put` operations on S3.